### DISCUSSION TITLE
Glances not recognized

### DISCUSSION THREAD

**Discussion by Wjpatx** [nicolargo/glances#2039] [General]
Hi,
I've installed glances via 'pip install glances' and when trying to run glances Cmd tells me it's not recognized as an internal or external command. I'm completely lost here and can't figure out why it's having this issue after installing just as the documentation asks. Am I just missing something completely?

**Comment by RazCrimson** [comment]
@Wjpatx 

Could you give more details about your system and how you tried to execute your glances (specifically the shell commands)?

**Reply by Wjpatx** [reply]
Windows 11
Python 3.9.12 ( tried as well with 3.10 same issue)

In CMD(admin moded) I just type `glances` as it says in the quickstart [guide](https://glances.readthedocs.io/en/latest/quickstart.html) and that's when i get the error.
`'glances' is not recognized as an internal or external command,
operable program or batch file.`

**Comment by Infinitay** [comment]
I am getting the same issues on my PC.
OS: `Windows 10`
Python Version: `3.9.9`
pip Version: `22.1`

```
PowerShell 7.2.6
Copyright (c) Microsoft Corporation.

https://aka.ms/powershell
Type 'help' to get help.

PS C:\Users\redacted\Documents\Code\Python\stable-diffusiont> python -m pip install --upgrade glances
Requirement already satisfied: glances in c:\python39\lib\site-packages (3.2.7)
Requirement already satisfied: defusedxml in c:\python39\lib\site-packages (from glances) (0.7.1)
Requirement already satisfied: psutil>=5.3.0 in c:\users\redacted\appdata\roaming\python\python39\site-packages (from glances) (5.8.0)
Requirement already satisfied: future in c:\python39\lib\site-packages (from glances) (0.18.2)
Requirement already satisfied: packaging in c:\python39\lib\site-packages (from glances) (21.3)
Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in c:\python39\lib\site-packages (from packaging->glances) (3.0.6)
PS C:\Users\redacted\Documents\Code\Python\stable-diffusiont> python -m pip install --upgrade psutil
Requirement already satisfied: psutil in c:\users\redacted\appdata\roaming\python\python39\site-packages (5.8.0)
Collecting psutil
  Using cached psutil-5.9.1-cp39-cp39-win_amd64.whl (245 kB)
Installing collected packages: psutil
  Attempting uninstall: psutil
    Found existing installation: psutil 5.8.0
    Uninstalling psutil-5.8.0:
      Successfully uninstalled psutil-5.8.0
Successfully installed psutil-5.9.1
PS C:\Users\redacted\Documents\Code\Python\stable-diffusiont> glances
glances: The term 'glances' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
PS C:\Users\redacted\Documents\Code\Python\stable-diffusiont> python --version
Python 3.9.9
PS C:\Users\redacted\Documents\Code\Python\stable-diffusiont> python -m pip --version
pip 22.1 from C:\Python39\lib\site-packages\pip (python 3.9)
```

FWIW I took a look within my Python installation/path (`C:\Python39\Scripts`) and I was expecting an executable for Glances but there was none.

**Reply by Infinitay** [reply]
Found a solution. When attempting to run glances as a module (?) via `python -m glances` I was finally able to run it. However, I was met with the following error:

> Curses module not found. Glances cannot start in standalone mode.
For Windows you can try installing windows-curses with pip install.

I then installed `windows-curses` via `python -m pip install windows-curses` and attempted to run glances again by specifying it as a module (`python -m glances` and it worked. Still can't run it _standalone_ via `glances` though.

@Wjpatx Give this a try, hopefully it fixes your problem as it did mine.
